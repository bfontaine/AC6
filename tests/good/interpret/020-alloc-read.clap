val x = alloc 42
val y = read x

