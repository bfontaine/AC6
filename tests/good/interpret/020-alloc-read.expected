:- x = <primitive>
:- y = 42
